### DISCUSSION TITLE
Python package psutil import failure

### DISCUSSION THREAD

**Discussion by oddwatcher** [termux/termux-packages#12172] [Q&A] [answered]
ImportError: dlopen failed: cannot locate symbol "ethtool_cmd_speed" referenced by "/data/data/com.termux/files/usr/lib/python3.10/site-packages/psutil/_psutil_linux.cpython-310.so"
when i tried to import psutil to python script
this also prevent me to use jupyter notebook, the kernel cannot start

**Comment by licy183** [comment, marked as answer]
See https://github.com/giampaolo/psutil/commit/b88f4d839dc4d665526e91b098d018a3d796a0ac and https://github.com/giampaolo/psutil/issues/2138. You could try to use `pip install psutil@git+https://github.com/giampaolo/psutil.git@b88f4d8`.

**Reply by xtkoba** [reply]
Version 5.9.3 includes the fix mentioned in the above comment.
